views/admin/rename_edit.tt

```diff
@@ -13,16 +13,11 @@ the same terms as Perl itself.  For a copy of the license, please reference
 %]
 
 [%- sections.title = "Detailed Rename View"; -%]
-[%- dw.need_res( 'stc/rename.css' ) -%]
+[%- CALL dw.active_resource_group( "foundation" ) -%]
 
-[% IF error_list %]
-    <p>Error</p>
-    <ul class='error-list'>
-        [% FOREACH error = error_list %]
-            <li>[% error %] </li>
-        [% END %]
-    </ul>
-[% END %]
+[%- dw.need_res( { group => "foundation" }
```

Comment:
I can't recall off-hand, but is there really supposed to be no trailing comma here?